File: HackerRank/30-Days-of-Code/C++/day01-DataTypes.cpp
#include <iomanip>
#include <iostream>
using namespace std;

//Question Link : https://www.hackerrank.com/challenges/30-data-types/problem?isFullScreen=true
int main() {
    int i=4;
    double d=4.0;
    string s="HackerRank ";
    int i2;
    double d2;
    string s2;
    string s3;
    cin>>i2;
    cin>>d2;
    getline(cin,s2);
    getline(cin,s3);
    double d3=d+d2;
    cout<<i+i2<<endl;
    cout<<fixed<<setprecision(1)<<d3<<endl;
    cout<<s+s3;  
    return 0;
}

File: HackerRank/30-Days-of-Code/Java/day01-DataTypes.java
import java.util.Scanner;
//Question Link : https://www.hackerrank.com/challenges/30-data-types/problem?isFullScreen=true

public class Solution {
	
    public static void main(String[] args) {
        int i = 4;
        double d = 4.0;
        String s = "HackerRank ";
		
        Scanner scan = new Scanner(System.in);

        int i2=scan.nextInt();
        double d2=scan.nextDouble();
        String s2=scan.nextLine();
        String s3=scan.nextLine();
        System.out.println(i+i2);
        System.out.println(d2+d);
        System.out.print(s+s3);

       
        scan.close();
    }
}